django/db/backends/sqlite3/base.py

```diff
@@ -297,7 +314,11 @@ def _start_transaction_under_autocommit(self):
         Staying in autocommit mode works around a bug of sqlite3 that breaks
         savepoints when autocommit is disabled.
         """
-        self.cursor().execute("BEGIN")
+        transaction_mode = self.settings_dict["OPTIONS"].get("transaction_mode")
+        if transaction_mode is None:
+            self.cursor().execute("BEGIN")
+        else:
+            self.cursor().execute(f"BEGIN {transaction_mode.upper()}")
```

Comment:
We don't allow calling methods in f-strings. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).